jobs/keepalived/templates/bin/ha_health_check.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
bin folders are usually not for source codes. If needed, keep this under src and package it during packaging.

Reply:
this change is now implemented. The new structure looks like this:
We have a src folder in our bosh-release. It will have 2 directories: `iaas-utils` and `ha-health-check`. The `ha-health-check` directory will contain this js source file.
These two directories will be referred to by the package: ha-utils.